app/models/order.py

```diff
@@ -23,7 +23,7 @@ def get_updatable_fields():
     """
     return ['country', 'address', 'city', 'state', 'zipcode', 'company', 'tax_business_info', 'status', 'paid_via',
             'order_notes', 'deleted_at', 'user', 'payment_mode', 'event', 'discount_code_id', 'discount_code',
-            'ticket_holders', 'user', 'tickets_pdf_url']
+            'ticket_holders', 'user', 'tickets_pdf_url', 'is_billing_enabled']
```

Comment:
this has to be conditionally updatable, if the order of status is complete, user should no longer be able to update it using the APIs, create this check on the before_update hook of orders api, and throw an exception with a pointer to this field, stating that billing options cannot be altered for an order which is complete.

Reply:
@CosmicCoder96 I've tried to fix it and pushed the updated code. Can you please tell me how to test it? I mean, when I try to order tickets, all the cases works fine i.e. when `is_billing_enabled` is true and `false`. Anything else, which is to be tested?